src/lib.rs

```diff
@@ -257,7 +250,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> HotShot<TYPES::ConsensusType
             high_qc: anchored_leaf.get_justify_qc(),
 
             metrics: Arc::new(ConsensusMetrics::new(
-                inner.metrics.subgroup("consensus".to_string()),
+                &*inner.metrics.subgroup("consensus".to_string()),
```

Comment:
For my own knowledge, why is this change better than the old version?

Reply:
The old version consumed the `Box` in the callee, which prevents the caller from ever using it again. This is unnecessary because the callee doesn't actually take ownership of the `Box`, it just borrows and then drops it.

In this particular case it doesn't matter because the caller is simply creating a temporary `Box` and then dropping it when the callee is done. But it's good practice for APIs to take objects by reference if they don't require ownership of the object.